dojson/utils.py

```diff
@@ -13,6 +13,25 @@
 import six
 
 
+class IgnoreKey(Exception):
+
+    """The corresponding key has been ignored."""
+
+    pass
+
+
+def ignore_value(f):
```

Comment:
@jma shall we accept optional argument with value that should be ignored otherwise we can rename it to `ignore_none_value`.